symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/ActivityRegistry.java

```diff
@@ -28,9 +31,37 @@
   /** The bot session forwarded to command-based activities only */
   private final UserV2 botSession;
 
+  private final MessageService messageService;
+
+  private SlashCommand helpCommand;
+
   /** The Datafeed real-time events source, or Datafeed listener */
   private final Consumer<RealTimeEventListener> realTimeEventsSource;
 
+  public ActivityRegistry(UserV2 botSession, Consumer<RealTimeEventListener> realTimeEventsSource , MessageService messageService) {
+    this.botSession = botSession;
+    this.realTimeEventsSource = realTimeEventsSource;
+    this.messageService = messageService;
+
+    this.registerHelpCommand();
+  }
+
+  private void registerHelpCommand() {
+    this.helpCommand = SlashCommand.slash("/help", commandContext -> {
+      List<String> commands = new ArrayList<>();
+      for (AbstractActivity<?, ?> activity : activityList) {
+        ActivityInfo info = activity.getInfo();
+        if (info instanceof CommandActivityInfo) {
+          CommandActivityInfo commandInfo = (CommandActivityInfo) info;
+          commands.add("<li>" + commandInfo.commandName() + ": " + commandInfo.summary() + "</li>");
+        }
+      }
+      String message = "<ul>" + String.join("\n", commands) + "</ul>";
+      this.messageService.send(commandContext.getStreamId(), Message.builder().content(message).build());
+    }, "Bdk Help Command");
```

Comment:
Is the help command listed in... help command result?

Reply:
yes it is, do we really need to exclude it?

Reply:
Maybe not, could be done later if necessary

Reply:
I was going to say we should not list it but then I checked what `ls --help` does for instance and it does list the --help arg so I guess it is ok to have it

Bdk-> BDK? (or maybe just `List available commands` ?)